core/src/main/java/discord4j/core/object/entity/channel/ForumChannel.java

```diff
@@ -19,125 +22,8 @@
 /**
  * A Discord guild channel that contains organized threads with labels.
  */
-@Experimental
```

Comment:
i prefer keep this until a stable release (?

Reply:
This is what 3.3.0 is intended, no?

Reply:
well yes in teory.